ooniapi/services/oonirun/src/oonirun/routers/v2.py

```diff
@@ -33,16 +37,26 @@
 def utcnow_seconds():
     return datetime.now(timezone.utc).replace(microsecond=0)
 
+class OonirunMeta(BaseModel):
+    run_type : str = Field(description="Run type", pattern="^(timed|manual)$")
+    is_charging : bool = Field(description="If the probe is charging")
+    probe_asn : str = Field(pattern=r"^([a-zA-Z0-9]+)$")
+    probe_cc : str = Field(description="Country code. Ex: VE")
+    network_type : str = Field(description="Ex: wifi")
```

Comment:
We also have an explicit list of valid `network_type`. @aanorbel @sdsantos can you provide us this list so we can add it here to the validation list?